### PR TITLE
Rename TaskResult to TaskRun

### PR DESCRIPTION
**Pull request by jhthompson**
Following up on https://github.com/RealOrangeOne/django-tasks/discussions/19, I figured I would go through and try renaming things to see how it feels in comparison (as per the [Django docs](https://docs.djangoproject.com/el/5.1/internals/contributing/bugs-and-features/#requesting-features), "code talks").

I don't love `TaskRunStatus`, it might be better as just `TaskStatus`, but that loses the association with the `TaskRun`/`TaskResult`/whatever else it might end up being named.

Mainly just opening to see what everyone thinks!

### CODE REVIEW

**Comment by hooverdc** [comment]
`TaskRunStatus` is a bit wordy, but overall I still find it easier to reason about than `TaskResult`.

**Comment by jhthompson** [comment]
Closing as the naming is going in another direction (per #19)
